jgrapht-core/src/main/java/org/jgrapht/alg/shortestpath/YenKShortestPaths.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * (C) Copyright 2017, by Keve Müller.

```

Comment:
Please use the original copyright statement as included in all files. The first copyright line should read:
* (C) Copyright 2017-2017, by Keve Müller and Contributors.

We use automated scripts to update the years, and these scripts don't like deviations in the copyright statement...